CHANGELOG.md

```diff
@@ -41,6 +41,7 @@
 - `postgresqlreceiver`: Fix issue where empty metrics could be returned after failed connection (#7502)
 - `resourcetotelemetry`: Ensure resource attributes are added to summary
   and exponential histogram data points. (#7523)
+- `prometheusexecreceiver`: Test zero value for `scrape_timeout` (#7594)
```

Comment:
This wouldn't be relevant for end-users. We have a label we can use to make CI ignore the changelog check, I'll set it to this PR.

Reply:
I dropped the commit and let you define the label